doc/toolbox-create.1.md

```diff
@@ -4,48 +4,144 @@
 toolbox\-create - Create a new toolbox container
 
 ## SYNOPSIS
-**toolbox create** [*--candidate-registry*]
```

Comment:
I split the removal of `--candidate-registry` into https://github.com/containers/toolbox/pull/662